lib/grizzly/zwave/command_classes.ex

```diff
@@ -14,33 +14,22 @@ defmodule Grizzly.ZWave.CommandClasses do
           secure_controlled: list(atom())
         ]
 
-  mappings = command_class_mappings()
-
-  command_classes_union =
-    mappings
-    |> Enum.map(&elem(&1, 1))
-    |> Enum.reverse()
-    |> Enum.reduce(&{:|, [], [&1, &2]})
-
-  @type command_class :: unquote(command_classes_union)
+  @type command_class :: atom()
 
   @doc """
   Get the byte representation of the command class
   """
   @spec to_byte(command_class()) :: byte()
-  for {byte, command_class} <- mappings do
-    def to_byte(unquote(command_class)), do: unquote(byte)
+  def to_byte(command_class) do
+    Map.fetch!(command_class_mappings(), command_class)
```

Comment:
Just curious, is there any difference when calling these functions from the module rather than importing them?